docs/docs/clients/overview.md

```diff
@@ -350,5 +350,3 @@ are all computed locally.
 - [Analytics-based Integrations](/integrations/overview#analytics-platforms) do not run.
   [Flag Analytics](/advanced-use/flag-analytics) do still work, if enabled within the
   [SDK setup](/clients/server-side#configuring-the-sdk).
-- In circumstances where you need to target a specific identity, you can do this by creating a segment to target that
-  specific user and subsequently adding a segment override for that segment.
```

Comment:
If we're removing this, shouldn't we also remove this line: https://github.com/Flagsmith/flagsmith/pull/3766/files#diff-a4321d583bb04a398f14c6f6164fe85200d8687ac5c617f8e52b631376463809L349 

... but should we also add some data about the Edge API opt-in process here?

Reply:
I've removed Ben's commit as we should really update the docs as part of FE work after #3881.